db/schema.rb

```diff
@@ -28,6 +28,19 @@
     t.datetime "updated_at"
   end
 
+  create_table "pieces", force: true do |t|
+    t.string   "piece_type"
+    t.integer  "active"
+    t.text     "square_location"
+    t.integer  "game_id"
+    t.integer  "player_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+    t.integer  "pieces_id"
```

Comment:
I don't think `pieces_id` is needed for anything.

Rails will automatically add a `id` column when creating this table. :smile: